packages/server/PWAHandler.ts

```diff
@@ -1,8 +1,10 @@
 import {HttpResponse, HttpRequest} from 'uWebSockets.js'
 import serveStatic from './utils/serveStatic'
 import acceptsBrotli from './acceptsBrotli'
+import safetyPatchRes from './safetyPatchRes';
 
 const PWAHandler = (res: HttpResponse, req: HttpRequest) => {
+  safetyPatchRes(res)
```

Comment:
this is interesting. you remember why this was required?

Reply:
I remember adding the safetyPatch because I was getting a fatalError when running Cypress. However, that was around the time we were both looking at that serveStatic issue, and not having the safetyPatch doesn't seem to give me an error anymore. I'll go ahead and remove it.